pkg/signal/trap.go

```diff
@@ -77,5 +79,30 @@ func DumpStacks() {
 	buf = buf[:stackSize]
 	// Note that if the daemon is started with a less-verbose log-level than "info" (the default), the goroutine
 	// traces won't show up in the log.
-	logrus.Infof("=== BEGIN goroutine stack dump ===\n%s\n=== END goroutine stack dump ===", buf)
+	if runtime.GOOS != "windows" {
```

Comment:
My line of thought was that it's a package that now behaves different, based on platform. Perhaps it's ok, idk